admin/controllers/administrator.js

```diff
@@ -0,0 +1,104 @@
+const Settings = require('../models/setting')
+const SettingsLog = require('../models/setting-log')
+const settingsErrorMessages = require('../lib/errors/settings')
+const settingsValidator = require('../lib/validator/settings-validator')
+const config = require('../config')
+
+const getAdministration = async (req, res, next) => {
+  res.locals.pageTitle = 'MTC Administration Homepage'
+
+  try {
+    req.breadcrumbs(res.locals.pageTitle)
+    res.render('administrator/administration-home', {
+      breadcrumbs: req.breadcrumbs()
+    })
+  } catch (error) {
+    next(error)
+  }
+}
+
+const getUpdateTiming = async (req, res, next) => {
+  res.locals.pageTitle = 'Check settings'
+  let settings
+  const successfulPost = req.params.status || false
+
+  try {
+    const settingsRecord = await Settings.find().exec()
```

Comment:
There is only one document in this collection, AFAIK we are expecting more fields (further settings) but not further documents so 'find' is meant to find the only document available (that is, the first one found, or [0])